docs/reproducing-results.md

```diff
@@ -45,11 +45,11 @@ documentation to learn how to make inference more efficient.
    trtllm-build \
       --checkpoint_dir /tmp/trtllm \
       --output_dir /code/openmath-mistral-7b-trtllm \
-      --gpt_attention_plugin bfloat16 \
-      --gemm_plugin bfloat16 \
-      --context_fmha "enable" \
-      --max_input_len 4096 \
-      --max_output_len 512 \
+      --gpt_attention_plugin bfloat16
+      --use_paged_context_fmha enable
+      --max_input_len 3584 \
+      --max_seq_len 4096 \
+      --max_num_tokens 4096 \
```

Comment:
Maybe a good idea to add a pointer in this doc to the original TRTLLM script?

Reply:
There is a pointer to their repo in the checkpoint conversion doc, this one is just for reproducing our results, so it seems proper to just include the command there that we used.

In general, there is unfortunately not really any good documentation page in trtllm repo, so we can't link to something more specific than just their github repository